game/common/player.py

```diff
@@ -5,7 +5,7 @@
 
 
 class Player(GameObject):
-    def __init__(self, code: object | None = None, team_name: str | None = None, action: ActionType | None = None,
+    def __init__(self, code: object | None = None, team_name: str | None = None, actions: list[ActionType] | list = [],
```

Comment:
actions: list[ActionType] | list = []

Having `| list ` before the = [] makes it so that it could also be an untyped list. I'm not sure if that was intentional or just a misunderstanding of default values.

Reply:
relevance repeated at references of this variable.

Reply:
Sleep deprived coding. Thank you for catching that. Sorry Caesar!